MetadataExtractor/Formats/Tiff/DirectoryTiffHandler.cs

```diff
@@ -46,6 +46,21 @@ protected DirectoryTiffHandler([NotNull] List<Directory> directories, [NotNull]
             Directories.Add(CurrentDirectory);
         }
 
+        protected bool ResetInitialDirectory([NotNull] Directory initialDirectory)
```

Comment:
Is there a better way to set the initial directory conditional on the TIFF marker? This feels like a workaround to a problem in the current design.